WallsAndHoles/tilemap.h

```diff
@@ -11,22 +11,16 @@
 #include <QSharedPointer>
 #include <QMutex>
 
-struct Properties
-{
-    bool outdoorMap = 0;
```

Comment:
Don't need to store both an outdoorMap and indoorMap, as they are mutually exclusive (one being false implies the other is true).